Source/WebKit/UIProcess/RemoteLayerTree/mac/ScrollingTreeOverflowScrollingNodeRemoteMac.cpp

```diff
@@ -63,11 +63,14 @@ void ScrollingTreeOverflowScrollingNodeRemoteMac::repositionRelatedLayers()
     m_delegate->updateScrollbarLayers();
 }
 
+void ScrollingTreeOverflowScrollingNodeRemoteMac::handleWheelEventPhase(const PlatformWheelEventPhase phase)
+{
+    m_delegate->handleWheelEventPhase(phase);
+}
+
 WheelEventHandlingResult ScrollingTreeOverflowScrollingNodeRemoteMac::handleWheelEvent(const PlatformWheelEvent& wheelEvent, EventTargeting eventTargeting)
 {
-    auto result = ScrollingTreeOverflowScrollingNodeMac::handleWheelEvent(wheelEvent, eventTargeting);
-    m_delegate->handleWheelEventForScrollbars(wheelEvent);
-    return result;
+    return ScrollingTreeOverflowScrollingNodeMac::handleWheelEvent(wheelEvent, eventTargeting);
```

Comment:
Can you just remove this override implementation?

Reply:
Fixed.